voorhoede/head-start
# fix: params.locale fallback on [locale]/ route mismatch

# Changes

404 route didn't have Astro.params.locale defined, causing problems in projects that rely on it, like in Internal Link.
This PR fixes that issue on a middleware level.

# Associated issue

N/A

# How to test

1. Open preview link
2. Navigate to /nl/aapje/ and verify locale is set to NL
3. Navigate to /en/aapje/ and verify locale is set to EN
4. Navigate to /aapje/ and verify locale is set to EN

# Checklist

- ~~I have made updated relevant documentation files (in project README, docs/, etc)~~
- ~~I have added a decision log entry if the change affects the architecture or changes a significant technology~~
